config/_default/menus.toml

```diff
@@ -69,42 +69,57 @@ weight = 4
 
 # footer menu left
 [[footer_left]]
-name = "Why Incubyte"
-url = "https://incubyte.co/why-incubyte"
+name = "home"
```

Comment:
Are we using lowercase links for footer?
Shouldn't we use "Home" instead of "home"?

Reply:
we are using lowercase for all links in the footer